modules/mux/targets/host/source/command_buffer.cpp

```diff
@@ -129,10 +129,10 @@ void populatePackedArgs(
             break;
         }
 
-        uint32_t sampler_ptr = libimg::HostCreateSampler(
+        uint64_t sampler_ptr = libimg::HostCreateSampler(
             info.sampler.normalize_coords, addressing_mode, filter_mode);
-        std::memcpy(packed_args_alloc + offset, &sampler_ptr, sizeof(uint32_t));
-        offset += sizeof(uint32_t);
+        std::memcpy(packed_args_alloc + offset, &sampler_ptr, sizeof(void *));
+        offset += sizeof(void *);
```

Comment:
since we tend towards size_t elsewhere - maybe we should use that here?

Reply:
Yeah, I was copying some lines from further up which were `void *`. But `void *` for pointers and `size_t` for values makes sense to me.